### PR TITLE
 Add parameter to give tables created by prepare step a two week TTL [VS-597]

### PR DESCRIPTION
**Pull request by rsasch**
See run [here](https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/26bfc19e-a804-4d68-9c1e-fb08c491e238) (and [here](https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/17e15516-fa1d-483e-96ee-038ea4b4eaad), after PR changes) and then the tables created in `gvs-internal.rsa_gvs_quickstart_dev`.

### CODE REVIEW

**Comment by rsasch** [comment]
> Looks good. Have you run the integration tests

I didn't run the integration test because it didn't seem necessary (no changes that effect the data, just the table(s) TTL). If you think I should, happy to kick it off.

File: scripts/variantstore/docs/aou/AOU_DELIVERABLES.md [hunk 1]
```diff
@@ -79,6 +79,7 @@
     - You will need to run `GvsPrepareRangesCallset` workflow first, if it has not been run already
        - This workflow transforms the data in the vet tables into a schema optimized for callset stats creation and for calculating sensitivity and precision.
        - The `only_output_vet_tables` input should be set to `true` (the default value is `false`).
+       - The `enable_extract_table_ttl` input should be set to `true` (the default value is `false`), which will add a TTL to the tables it creates.
```
Comment (RoriCremer) [inline scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:82]: TTL makes sense to me in this context, but do we think it should be defined here?

Comment (RoriCremer) [reply scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:82]: and I dont know where this should live, but we set the default to 2 weeks in the wdl? do we want to document that anywhere?

**Review by RoriCremer — APPROVED** [review]

File: scripts/variantstore/docs/aou/AOU_DELIVERABLES.md (hunk 1 above)
Comment (rsasch) [reply scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:82]: I will add that it's two weeks here.

**Pull request merged**